nixos/modules/system/boot/loader/grub/grub.nix

```diff
@@ -224,7 +224,11 @@ in
 
       extraConfig = mkOption {
         default = "";
-        example = "serial; terminal_output.serial";
+        example = ''
+          serial --unit=0 --speed=115200 --word=8 --parity=no --stop=1
+          terminal_input --append serial
+          terminal_output --append serial
```

Comment:
Is `terminal_input console` `terminal_output console` the default?

I usually have 
```
terminal_input serial console; terminal_output serial console
```
so I wonder abotu the `--append` here…

Reply:
I was looking at that as well; my understanding from https://www.gnu.org/software/grub/manual/grub/html_node/terminal_005finput.html is that it is good because with `--append` you can say "also expose it to serial, no matter whether `gfxterm` or `console` is the main output".

I haven't actually tried that yet though, that is, I haven't checked whether the combination of `gfxterm serial` actually works as expected.

Reply:
> I haven't actually tried that yet though, that is, I haven't checked whether the combination of `gfxterm serial` actually works as expected.

I have checked it now, it works.